integration-tests/api/__tests__/admin/product.js

```diff
@@ -1456,6 +1471,96 @@ describe("/admin/products", () => {
         })
       )
     })
+
+    it("updates a product's categories", async () => {
+      const api = useApi()
+      const category = await simpleProductCategoryFactory(
```

Comment:
suggestion(non blocking): Same as usual, when using the id at different places it is better to store it in a variable. It is easier to update if needed